src/schema/filter_artworks.js

```diff
@@ -97,24 +96,21 @@ export const FilterArtworksType = new GraphQLObjectType({
         { options: gravityOptions },
         args,
         request,
-        { rootValue: { accessToken } }
+        { rootValue: { filterArtworksLoader } }
       ) => {
         const relayOptions = parseRelayOptions(args)
-        return gravity
-          .with(accessToken)(
-            "filter/artworks",
-            assign(gravityOptions, relayOptions, {})
-          )
-          .then(({ aggregations, hits }) => {
-            if (!aggregations || !aggregations.total) {
-              throw new Error("This query must contain the total aggregation")
-            }
+        return filterArtworksLoader(
+          assign(gravityOptions, relayOptions, {})
```

Comment:
Not code you changed, but noticing this now. The order of arguments is probably incorrect. I assume the intention was to merge `gravityOptions` and `relayOptions` into the empty object, but the destination object should be the first argument. Also there’s the option nowadays to do `{ ...gravityOptions, ...relayOptions }`.